src/Symfony/Component/ObjectMapper/Metadata/ReflectionObjectMapperMetadataFactory.php

```diff
@@ -18,12 +18,17 @@
  * @internal
  *
  * @author Antoine Bluchet <soyuka@gmail.com>
+ * @author Florent Blaison <florent.blaison@gmail.com>
  */
 final class ReflectionObjectMapperMetadataFactory implements ObjectMapperMetadataFactoryInterface
 {
     private array $reflectionClassCache = [];
     private array $attributesCache = [];
 
+    public function __construct(private readonly array $classMap)
```

Comment:
```suggestion
    public function __construct(private readonly array $classMap = [])
```

should have a default value for bc layer